lib/que/web.rb

```diff
@@ -185,6 +185,7 @@ def format_error(job)
       end
 
       def relative_time(time)
+        time = Time.utc((t = Date._parse(time))[:year], t[:mon], t[:mday], t[:hour], t[:min], t[:sec]) if time.is_a?(String)
```

Comment:
What about using [Time::parse](https://ruby-doc.org/stdlib-2.7.2/libdoc/time/rdoc/Time.html#method-c-parse) from stdlib for parsing to a time like your original proposal?

```ruby
# Top of file
require 'time'


def relative_time(time)
  time = Time.parse(time) if time.is_a?(String)
  %{<time class="timeago" datetime="#{time.utc.iso8601}">#{time.utc}</time>}
end
```

Reply:
`Time.parse()` parses using local time with the result that
```
time = '2020-09-06 22:17:03'
relative_time(time)
```
returns `<time class="timeago" datetime="2020-09-07T02:17:03Z">2020-09-07 02:17:03 UTC</time>`
for my current timezone (EDT).

However, I believe `run_at` is not meant to be treated as local.

When I add a job with a `run_at` in utc (example `1.year.from_now.utc` => Fri, 29 Oct 2021 20:43:40 UTC +00:00), run_at is returned as "2021-10-29 20:30:01.405839" using Que::Web::SQL[:scheduled_jobs]

When I add a job with `run at` in local time (example `Time.now + 1.year` => 2021-10-29 16:44:14 -0400), , run_at is returned as "2021-10-29 20:44:14.053852" again using Que::Web::SQL[:scheduled_jobs]

In my setup of Postgres, pg, Rails and que, I think `run_at` is returned as a string without timezone and is assumed to be in UTC, not local.

Reply:
```
time = Time.use_zone('UTC') { Time.zone.parse time } if time.is_a?(String)
```
might be more expressive and clearer.